src/web/clients/api/rate-limited.md

```diff
@@ -69,5 +69,5 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
-[`hyper::header!`]: https://docs.rs/hyper/*/hyper/macro.header.html
+[`hyper::header!`]: https://doc.servo.org/hyper/header/index.html#defining-custom-headers

```

Comment:
This basically just references the 0.11 branch without breaking the wildcard rule.  Not ideal, but #419 should solve it.